deployments/ansible/group_vars/all

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Is this some kind of defaults file or just for testing?

Reply:
'all' is default file in Ansible which defines common variables across all types of hosts. And, there will be group specific var files as siblings if any, which contains variables specific to that group of hosts, like 'webservers', 'dbservers', etc.